site/content/blog/2022-08-01-whats-new-in-svelte-august-2022.md

```diff
@@ -0,0 +1,119 @@
+- The `prepare` script is no longer needed in `package.json` ([#5760](https://github.com/sveltejs/kit/pull/5760))
+- `adapter-node` no longer does any compression while we wait for a [bug fix in the `compression` library](https://github.com/expressjs/compression/pull/183) ([#5560](https://github.com/sveltejs/kit/pull/5506))
+
+For a full list of changes, check out kit's [CHANGELOG](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md).
+
+
+## What's new in Svelte & Language Tools
+- The `@layer` [CSS at-rule](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer) is now supported in Svelte components (**3.49.0**, [PR](https://github.com/sveltejs/svelte/issues/7504))
+- The `inert` [HTML attribute](https://html.spec.whatwg.org/multipage/interaction.html#the-inert-attribute) is now supported in Svelte's language tools and plugins ( **105.20.0**, [PR](https://github.com/sveltejs/language-tools/pull/1565))
+- The Svelte plugin will now use `SvelteComponentTyped` typings, if available (**105.19.0**, [PR](https://github.com/sveltejs/language-tools/pull/1548))
+
+
+---
+
+## Community Showcase
+
+**Apps & Sites built with Svelte**
+- [PocketBase](https://github.com/pocketbase/pocketbase) is an open source Go backend with a single file and an admin dashboard built with Svelte
+- [Hondo](https://www.playhondo.com/how-to-play) is a word guessing game with multiple rounds
+- [Hexapipes](https://github.com/gereleth/hexapipes) is a site for playing hexagonal pipes puzzle
+- [Mail Must Move](https://www.mordon.app/) is an email made for those who want to get more done
+- [Jot Down](https://github.com/brysonbw/vscode-jot-down) is a Visual Studio Code extension for quick and simple note taking
+- [Kadium](https://kadium.kasper.space/) is an app for staying on top of YouTube channels' uploads
+- [Samen zjin we #1metS10](https://1mets10.avrotros.nl/) is a campaign website to support S10, the dutch Eurovision finalist, by sending a drawing or a wish
+- [On Writing Code](https://onwritingcode.com/) is an interactive website to learn programming design patterns
+- [Svelte-In-Motion](https://github.com/novacbn/svelte-in-motion) lets you create Svelte-animated videos in your browser
+- [Svelte Terminal](https://github.com/Nico-Mayer/svelte-terminal) is a terminal-like website
+- [Bulletlist](https://bulletlist.com/) is a simple tool with a single purpose: making lists
+- [Remind Me Again](https://github.com/probablykasper/remind-me-again) is an app for toggleable reminders on Mac, Linux and Windows
+- [Heyweek](https://heyweek.com/) is a timetracking app built for freelancers craving that extra pizzazz
+
+**Learning Resources**
+
+_Starring the Svelte team_
+- [The Svelte Documentary is out!](https://www.svelteradio.com/episodes/the-svelte-documentary-is-out) on Svelte Radio
+- [Beginner SvelteKit](https://vercel.com/docs/beginner-sveltekit) by Vite
```

Comment:
```suggestion
- [Beginner SvelteKit](https://vercel.com/docs/beginner-sveltekit) by Vercel
```

Reply:
Steph is a Svelte Siren: https://sveltesirens.dev/about